source/MaterialXView/Viewer.cpp

```diff
@@ -1349,15 +1327,12 @@ void Viewer::initContext(mx::GenContext& context)
 void Viewer::loadStandardLibraries()
 {
     // Initialize the standard library.
-    _stdLib = loadLibraries(_libraryFolders, _searchPath);
-    if (_stdLib->getChildren().empty())
+    _stdLib = mx::createDocument();
+    _xincludeFiles = mx::loadLibraries(_libraryFolders, _searchPath, _stdLib);
+    if (_xincludeFiles.empty())
```

Comment:
@jstone-lucasfilm, This I believe would be a separate change -- as how to automatically exclude elements with these source URIs. As we used the common routine, there is not need to search for sourceURIs after the fact.